tests/staticfiles_tests/test_storage.py

```diff
@@ -244,6 +244,56 @@ def test_cache_key_memcache_validation(self):
         self.assertEqual(cache_key, 'staticfiles:821ea71ef36f95b3922a77f7364670e7')
 
 
+@override_settings(
+    **dict(
+        TEST_SETTINGS,
+        STATICFILES_STORAGE=(
+            'staticfiles_tests.storage.MultiCachedStaticFilesStorage'
+        ),
+        DEBUG=False,  # see note above
+    )
+)
+class TestMultiCachedStorage(BaseCollectionTestCase, BaseStaticFilesTestCase,
```

Comment:
This can probably also be cleaned in existing cases but `BaseCollectionTestCase` inherits `BaseStaticFilesTestCase` so it looks like it's redundant here.